driver-core/src/test/unit/com/mongodb/internal/connection/InternalStreamConnectionSpecification.groovy

```diff
@@ -278,12 +276,18 @@ class InternalStreamConnectionSpecification extends Specification {
 
     def 'should throw MongoInternalException when reply header message length > max message length'() {
         given:
-        stream.read(36, 0) >> { helper.headerWithMessageSizeGreaterThanMax(1) }
+        Debugger.OverridingReportingMode debuggerReportingMode = Debugger.useReportingMode(Debugger.ReportingMode.LOG)
```

Comment:
This and the below test both simulate a situation that the debugger is meant to react on. Here we are changing the default test reporting mode from `LOG_AND_THROW` to `LOG` to not disrupt the tests.